src/connectivity.jl

```diff
@@ -283,6 +283,204 @@ function strongly_connected_components end
     return components
 end
 
+"""
+    strongly_connected_components_kosaraju(g)
+
+Compute the strongly connected components of a directed graph `g` using Kosaraju's Algorithm. 
+(https://en.wikipedia.org/wiki/Kosaraju%27s_algorithm).
+
+Return an array of arrays, each of which is the entire connected component.
+
+### Performance
```

Comment:
Stating time and space complexity here is good,  but I think we should put the more detailed information in a comment somewhere else, as this is not relevant information when someone wants to quickly look up, what this function does.

Reply:
Shortened the information.